CMakeLists.txt

```diff
@@ -10,7 +10,23 @@ project(SQLiteCpp VERSION "2.99")
 
 # SQLiteC++ 3.x now requires C++11 compiler
 set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+option(USE_LATEST_CXX_STANDARD_VERSION "Makes CMake to probe for the latest CXX version. May give efficiency improvements. Disables mandating minimal standard version: CMake has design flaws that prevents from adding a minimal version constraint while opportunistically upgrading: CMAKE_CXX_STANDARD_REQUIRED should have been an int for that, not a boolean" OFF)
```

Comment:
Could you please rename it to USE_LATEST_CXX_STANDARD and make the description shorter by moving the note about CMake design flaw to a simple comment above the option

Reply:
this one is good now :)